components/popover/package.json

```diff
@@ -17,19 +17,19 @@
   },
   "peerDependencies": {
     "@spectrum-css/dialog": "^6.0.0",
-    "@spectrum-css/vars": "^8.0.0"
+    "@spectrum-css/tokens": "^1.0.4"
   },
   "devDependencies": {
     "@spectrum-css/button": "^6.0.13",
-    "@spectrum-css/component-builder": "^3.2.0",
+    "@spectrum-css/component-builder-simple": "^1.0.0-beta.0",
     "@spectrum-css/dialog": "^6.0.15",
     "@spectrum-css/divider": "^1.0.27",
     "@spectrum-css/menu": "^4.0.4",
-    "@spectrum-css/vars": "^8.0.0",
+    "@spectrum-css/tokens": "^1.0.4",
```

Comment:
same: I think we are now good to use version `1.0.7`

Reply:
Fixed in latest commit.